### PR TITLE
Fixes to the lost topics

### PR DESCRIPTION
**Pull request by alexandraepan**
### Description 
Fixes to the lost topics. 

Fixes # - _issue number(s) if exists_

- [x] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [x] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [x] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [x] not needed

### Breaks backward compatibility
- [ ] Yes
- [x] No
- [ ] Unknown

### Notify the following users
_List users with `@` to send notifications_

### CODE REVIEW

File: doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .. _malloc_replacement_log:
 
-oneTBB_malloc_replacement_log Function
-======================================
+TBB_malloc_replacement_log
+==========================
```
Comment (akukanov) [inline doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst:4]: The "Function" was appropriate in the page header. Sorry if my comment was misleading in this regard.

File: doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst [hunk 2]
```diff
@@ -81,4 +81,4 @@ Example output:
 
    tbbmalloc_proxy cannot replace memory allocation routines
    Success: free (ucrtbase.dll), byte pattern: <C7442410000000008B4424>
-   Fail: _msize (ucrtbase.dll), byte pattern: <E90B000000CCCCCCCCCCCC>
\ No newline at end of file
+   Fail: _msize (ucrtbase.dll), byte pattern: <E90B000000CCCCCCCCCCCC>
```
Comment (akukanov) [inline doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst:84]: What's changed here? Is it an accidental line break deletion?

File: doc/main/tbb_userguide/Floating_Point_Settings.rst [hunk 3]
```diff
@@ -32,19 +31,19 @@ You can then pass the task group context to most parallel algorithms, including
 It is possible to execute the parallel algorithms with different floating-point settings captured to separate contexts, even at the same time.
 
 Floating-point settings captured to a task group context prevail over the settings captured during task scheduler initialization. It means, if a context is passed to a parallel algorithm, the floating-point settings captured to the context are used. 
-Otherwise, if floating-point settings are not captured to the context, or a context is not explicitly specified, the settings captured during task scheduler initialization are used.
+Otherwise, if floating-point settings are not captured to the context, or a context is not explicitly specified, the settings captured during the task arena initialization are used.
 
 In a nested call to a parallel algorithm that does not use the context of a task group with explicitly captured floating-point settings, the outer-level settings are used. 
 If none of the outer-level contexts capture floating-point settings, the settings captured during task scheduler initialization are used.
```
Comment (akukanov) [inline doc/main/tbb_userguide/Floating_Point_Settings.rst:37]: Change here the "task scheduler" to the "task arena" as well, similarly to line 34

File: doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst (hunk 2 above)
Comment (alexandraepan) [reply doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst:84]: Nothing was changed there, not sure why it is displayed that way.

**Review by akukanov — APPROVED** [review]

**Pull request merged**